src/Router.tsx

```diff
@@ -2,13 +2,16 @@ import React from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import Home from './pages/Home/index';
 import Login from './pages/Login/index';
-
+import ShareTogether from 'pages/ShareTogether';
+import ShareTogetherDetail from 'pages/ShareTogetherDetail';
 function Router(): React.ReactElement {
   return (
     <BrowserRouter>
       <Switch>
         <Route exact path="/" component={Home} />
         <Route exact path="/Login" component={Login} />
+        <Route exact path="/ShareTogether" component={ShareTogether} />
+        <Route exact path="/ShareTogetherDetail/:id" component={ShareTogetherDetail} />
```

Comment:
여기 /로 들어가는 거 자체가 상세페이지 보는거니까
```suggestion
        <Route exact path="/ShareTogether" component={ShareTogether} />
        <Route exact path="/ShareTogether/:id" component={ShareTogether} />
```